package.json

```diff
@@ -10,13 +10,16 @@
   ],
   "repository": "git@github.com:distolma/storeon-observable.git",
   "author": "Dmytro Mostovyi <djmakaron2009@gmail.com>",
+  "contributors": [
+    "Dmytro Mostovyi <djmakaron2009@gmail.com>",
+    "Paweł Majewski <majo44@gmail.com>"
+  ],
   "license": "MIT",
   "scripts": {
     "start": "rollup -c rollup.config.ts -w",
-    "build": "tsc && tsc -p tsconfig.module.json",
+    "build": "tsc && tsc -p tsconfig.module.json && size-limit",
     "lint": "eslint index.ts",
-    "size": "size-limit",
-    "test": "jest --coverage && yarn lint && size-limit"
+    "test": "jest --coverage && yarn lint"
```

Comment:
Better run `build` before size-limit, imho
```sh
jest --coverage && yarn lint && npm build && size-limit
```

Reply:
I changed the scripts, so now `build` is main script which orchestrate all other